tools/converter/CMakeLists.txt

```diff
@@ -47,16 +47,9 @@ IF(MNN_BUILD_CONVERTER)
   
   IF(NOT MNN_BUILD_SHARED_LIBS)
     target_link_libraries(MNNConvertDeps PUBLIC ${Protobuf_LIBRARIES})
+    target_link_libraries(MNNConvertDeps PUBLIC z)
     if(APPLE)
-# Mac default zlib only have .dylib, but we need .a
-# So you should run `brew install zlib` (include .a)
-# Due to cmake find_package(zlib) try to detect .dylib even if set ZLIB_ROOT explicitly, so we need pkg-config
-# We need `export PKG_CONFIG_PATH=/usr/local/Cellar/zlib/{version}/lib/pkgconfig:$PKG_CONFIG_PATH`
-# Now we can omit zlib.a dependency by `pkg-config --libs zlib`
-        find_package(PkgConfig REQUIRED)
-        pkg_check_modules(ZLIB REQUIRED zlib)
-        string(REPLACE .dylib .a ZLIB_STATIC_LIBRARIES ${ZLIB_LINK_LIBRARIES})
-        target_link_libraries(MNNConvert -Wl,-all_load MNNConvertDeps ${MNN_DEPS} ${ZLIB_STATIC_LIBRARIES} -Wl,-noall_load)
+        target_link_libraries(MNNConvert -Wl,-all_load MNNConvertDeps ${MNN_DEPS} -Wl,-noall_load)
```

Comment:
The modify may cause compile error for build pymnn in mac. Could you help build it and test?